codegenerator/cli/templates/static/codegen/src/Utils.res

```diff
@@ -132,6 +132,14 @@ Helper to check if a value exists in an array
       [...head, ...tail]
     }
   }
+
+  let rec find = (arr, fn, ~index=0) => {
+    open Belt
+    switch arr[index] {
+    | Some(item) => item->fn ? item->Some : arr->find(fn, ~index=index + 1)
+    | None => None
+    }
+  }
```

Comment:
There should be `Js.Array2.find` or `Belt.Array.keepMap`

Reply:
keepMap will not exit early and return a whole array of matching items though.

I'm not sure of Js.Array2.find, that probably does exit early. If it's native implementation it's probably best.

Reply:
Yes, it was added in es6 and exits early